libspud/src/tests/test_main.cpp

```diff
@@ -38,12 +38,12 @@ extern "C"{
 
 int main(int argc, char **argv) {
 #ifdef HAVE_MPI
-  MPI_Init(&argc, &argv);
+  MPI::Init(argc, argv);
```

Comment:
This reverts the switch away from the C++ mph bindings. Which is fine, we just need to remember to back port the change upstream.